<launch>
  <arg name="camera"      default="camera"/>

  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch"/>
  <include file="$(find turtlebot_bringup)/launch/minimal.launch"/>

  <node pkg="nodelet" type="nodelet" name="tracker_nodelet" args="load hri_planner/TrackerNodelet $(arg camera)/$(arg camera)_nodelet_manager" output="screen">
	  <param name="camera_info_file" value="$(find hri_planner)/camera_calibration/kinect_rgb.yml"/>
	  <param name="map_file" value="$(find hri_planner)/maps/marker_maps/exp_test.json"/>
	  <param name="filter_param_file" value="$(find hri_planner)/params/robot_tracker.json"/>
	  <param name="marker_size" value="0.204"/>
	  <param name="filter_mode" value="kalman"/>
      <param name="odom_source" value="odometer"/>
	  <param name="num_frames_skip" value="2"/>
	  <param name="image_scale" value="1.0"/>
	  <remap from="/$(arg camera)/inside_out_tracker/image" to="/$(arg camera)/rgb/image_mono"/>
	  <remap from="/$(arg camera)/inside_out_tracker/pose2d" to="/tracking/robot_pose2d"/>
	  <remap from="/$(arg camera)/inside_out_tracker/odom" to="/odom"/>
  </node>

  <!--node pkg="hri_planner" type="imu_manager_node" name="imu_manager" output="screen">
	<remap from="/imu_data_raw" to="/tracking/imu_data_raw"/>
  </node-->
</launch>
